mongo-helm/templates/mongo.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongodb-deployment
+  labels:
+    app: {{ .Values.appName}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Values.appName}}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.appName}}
+    spec:
+      containers:
+      - name: {{ .Values.appName}}
+        image: {{ .Values.mongo.imageName }}:{{ .Values.mongo.imageTag }}
+        ports:
+        - containerPort: 27017
+        env:
+        - name: MONGO_INITDB_ROOT_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: mongodb-secret
+              key: mongo-root-username
+        - name: MONGO_INITDB_ROOT_PASSWORD
+          valueFrom: 
+            secretKeyRef:
+              name: mongodb-secret
+              key: mongo-root-password
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongodb-service
+spec:
+  selector:
+    app: {{ .Values.appName}}
+  ports:
+    - protocol: TCP
+      port: 27017
+      targetPort: 27017

```

Comment:
Please create a separate yaml file for this.

Reply:
Done.